bindata/network/ovn-kubernetes/self-hosted/ovnkube-master.yaml

```diff
@@ -382,7 +383,11 @@ spec:
                 echo "NB DB Raft leader is unknown to the cluster node."
                 exit 1
               else
-                /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null
+                # enable memory triming once
+                if [ ! -f /var/run/ovn/ovnnb_db_mem_trim.txt ]; then
+                  /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null
+                  touch /var/run/ovn/ovnnbdb-mem-trim-enabled
```

Comment:
Should also be a ` &&`
```shell
/usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null && touch /var/run/ovn/ovnnbdb-mem-trim-enabled
```
  of `if/then` so that we only touch on success.